build.gradle

```diff
@@ -46,8 +46,10 @@ task pmd(type: Pmd) {
   exclude '**/**.mp3'
   exclude 'test/bash/bats/**'
   dependencies {
-    pmd 'net.sourceforge.pmd:pmd-java:6.55.0'
-    pmd 'net.sourceforge.pmd:pmd-xml:6.55.0'
+    pmd 'net.sourceforge.pmd:pmd-cli:7.1.0'
+    pmd 'net.sourceforge.pmd:pmd-ant:7.1.0'
```

Comment:
Are these 2 required now? :O

Reply:
Yes they split up their code into different packages in v7.0 and these two are now required